core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -213,7 +213,9 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
     logInfo("Connecting to external H2O cluster.")
     val clusterBuildTimeout = hc.getConf.cloudTimeout
     val nodes = if (runningFromNonJVMClient(hc)) {
-      getNodes(hc._conf)
+      val nodes = getNodes(hc._conf)
```

Comment:
nit: shouldn't we be consistent and use `hc.getConf` everywhere?

Reply:
yup, thanks, good catch!